tests/setup_cfg_fmt_test.py

```diff
@@ -62,16 +62,65 @@ def test_noop(tmpdir):
             '\n'
             '[metadata]\n'
             'version = 1.0\n'
-            'name = pkg\n',
+            'name = pkg\n'
+            '[options]\n'
+            'install_requires =\n'
+            '    req3\n'
+            '    req5 <= 2,!=1\n'
+            '    req6 ;python_version==2.7\n'
+            '    req7 ;os_version!=windows\n'
```

Comment:
can you also show `req; python_version=="2.7"` normalized to `req;python_version=="2.7"`?

Reply:
sure

Reply:
Did you know this was going to be a bug ahead of time? :P

Reply:
yes